build.gradle

```diff
@@ -188,6 +188,7 @@ project(":samza-core_$scalaSuffix") {
     compile "org.scala-lang:scala-library:$scalaVersion"
     compile "org.slf4j:slf4j-api:$slf4jVersion"
     compile "net.jodah:failsafe:$failsafeVersion"
+    compile "org.scala-lang:scala-library:$scalaVersion"
```

Comment:
Why is this required?

Reply:
This could be a mistake when I rebased the branch with latest master.